autoware_map_msgs/README.md

```diff
@@ -4,25 +4,47 @@
 
 The message represents an area information. This is intended to be used as a query for partial / differential map loading (see `GetPartialPointCloudMap.srv` and `GetDifferentialPointCloudMap.srv` section).
 
-## PointCloudMapCellWithID.msg
+| field                 | detail | constraint | note |
+| --------------------- | ------ | ---------- | ---- |
+| `center_x`: `float32` | -      | -          | -    |
+| `center_y`: `float32` | -      | -          | -    |
+| `radius`: `float32`   | -      | -          | -    |
```

Comment:
Are you planning to handle the contents of the detail in a separate PR?

Reply:
Ops, this is still draft